OnlineFlightSearchAPI/Services/AirportServices/AirportServices.cs

```diff
@@ -0,0 +1,37 @@
+﻿using OnlineFlightSearchAPI.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OnlineFlightSearchAPI.FlightServices
+{
+    public class AirportServices : IAirportServices
+    {
+        private List<AirportDetail> airportDetails = new List<AirportDetail>();
```

Comment:
abstract the list out, may be in to a repository class? Will make it easier for switching the data sources

Reply:
Also, makes it easier for testing...

Reply:
Moved to Interface